hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -418,27 +657,81 @@ public:
    /// Normalized axes access, converting from actual axis type to base class
    const RAxisBase &GetAxis(int iAxis) const final { return *std::apply(Internal::GetAxisView<AXISCONFIG...>, fAxes)[iAxis]; }
 
-   /// Gets the bin index for coordinate `x`; returns -1 if there is no such bin,
+   /// 
+   std::vector<int> GetBinType(const CoordArray_t &x) const
+   {
+      return Internal::RGetOverflowBin<DATA::GetNDim() - 1, RHistImpl, decltype(fAxes)>()(nullptr, fAxes, x);
```

Comment:
Method name vs template implementation name is inconsistent (`GetBinType` vs `RGetOverflowBin`). Please pick one or the other and use it everywhere.